homeassistant/components/meater/__init__.py

```diff
@@ -0,0 +1,85 @@
+"""The Meater Temperature Probe integration."""
+import asyncio
+import logging
+
+from meater import MeaterApi
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = vol.Schema(
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Required(CONF_USERNAME): cv.string,
+                vol.Required(CONF_PASSWORD): cv.string,
+            }
+        )
+    },
+    extra=vol.ALLOW_EXTRA,
+)
```

Comment:
Please remove the YAML configuration possibility. It is not something we want to add to new integrations.

Reply:
Ah, sorry, I removed the references to YAML configuration in the docs but didn't realise we were removing the YAML configuration totally from new integrations. I'll fix now.